src/codelabs/async-await.md

```diff
@@ -42,6 +42,17 @@ another operation to finish. Here are some common asynchronous operations:
 To perform asynchronous operations in Dart, you can use the `Future` class
 and the `async` and `await` keywords.
 
+Typically, APIs that return `Future`s fall into one of the following categories:
+
+* APIs that are backed by logic running on a different thread. For example,
+  `File.readAsString` (and most of the other I/O APIs) call operating system
+  functions from threads on a dedicated thread pool, rather than from the
+  thread on which Dart code is exectured.
```

Comment:
Just adjusting the semantic line breaks and fixing the spelling of "executed".

```suggestion
* APIs that are backed by logic running on a different thread. 
  For example, `File.readAsString` (and most of the other I/O APIs) 
  call operating system functions from threads on a dedicated thread pool, 
  rather than from the thread on which Dart code is executed.
```